eegio/base/objects/bids_patient.py

```diff
@@ -0,0 +1,320 @@
+import os
+import sys
+import warnings
+import pickle
+import shutil as sh
+
+import mne
+from mne import io
+from mne_bids import write_raw_bids, make_bids_basename, make_bids_folders
+from mne_bids.utils import print_dir_tree
+
+from typing import List, Dict
+
+parentdir = os.path.join(os.getcwd(),'..')
+sys.path.append(parentdir)
+from config import EEGIO_DIR
```

Comment:
you should be able to just do from eegio.base.config import EEGIO_DIR

Not sure if you use Pycharm, or VisualStudio (they will show you import errors)